mobile/src/main/java/com/truckmuncher/truckmuncher/authentication/AuthenticatorActivity.java

```diff
@@ -64,4 +70,13 @@ public void onLoginSuccess(String userName, String authToken) {
         setResult(RESULT_OK, intent);
         finish();
     }
+
+    @Override
+    public void onActivityResult(int requestCode, int resultCode, Intent data) {
+        super.onActivityResult(requestCode, resultCode, data);
+
+        if (fragment != null) {
+            fragment.onActivityResult(requestCode, resultCode, data);
```

Comment:
I'm not sure exactly how the Facebook stuff works either, but that was what they said to do in their tutorial.